server/admin/api.go

```diff
@@ -107,6 +108,83 @@ func (s *Server) apiMarketInfo(w http.ResponseWriter, r *http.Request) {
 	writeJSON(w, mktStatus)
 }
 
+// apiMarketOrderBook is the handler for the '/market/{marketName}/orderbook'
+// API request.
+func (s *Server) apiMarketOrderBook(w http.ResponseWriter, r *http.Request) {
+	mkt := strings.ToLower(chi.URLParam(r, marketNameKey))
+	status := s.core.MarketStatus(mkt)
+	if status == nil {
+		http.Error(w, fmt.Sprintf("unknown market %q", mkt), http.StatusBadRequest)
+		return
+	}
+	orders, err := s.core.BookOrders(status.Base, status.Quote)
+	if err != nil {
+		http.Error(w, fmt.Sprintf("failed to obtain order book: %v", err), http.StatusInternalServerError)
+		return
+	}
+	msgBook := make([]*msgjson.BookOrderNote, 0, len(orders))
+	for _, o := range orders {
+		msgBook = append(msgBook, market.OrderToMsgOrder(o, mkt))
+	}
+	msg := &msgjson.OrderBook{
+		MarketID: mkt,
+		Epoch:    uint64(status.ActiveEpoch),
+		Orders:   msgBook,
+	}
+	writeJSON(w, msg)
+}
+
+// hander for route '/market/{marketName}/epochorders' API request.
+func (s *Server) apiMarketEpochOrders(w http.ResponseWriter, r *http.Request) {
+	mkt := strings.ToLower(chi.URLParam(r, marketNameKey))
+	status := s.core.MarketStatus(mkt)
+	if status == nil {
+		http.Error(w, fmt.Sprintf("unknown market %q", mkt), http.StatusBadRequest)
+		return
+	}
+	orders, err := s.core.EpochOrders(status.Base, status.Quote)
+	if err != nil {
+		http.Error(w, fmt.Sprintf("failed to obtain epoch orders: %v", err), http.StatusInternalServerError)
+		return
+	}
+	msgBook := make([]*msgjson.BookOrderNote, 0, len(orders))
+	for _, o := range orders {
+		msgBook = append(msgBook, market.OrderToMsgOrder(o, mkt))
+	}
+	msg := &msgjson.OrderBook{
+		MarketID: mkt,
+		Epoch:    uint64(status.ActiveEpoch),
+		Orders:   msgBook,
+	}
+	writeJSON(w, msg)
```

Comment:
Just noticed that we're leaving the `Seq` field unset always.  Let's keep it out of the admin api response with something like

```go
res := &struct {
	MarketID string                   `json:"marketid"`
	Epoch    uint64                   `json:"epoch"`
	Orders   []*msgjson.BookOrderNote `json:"orders"`
}{
	MarketID: mkt,
	Epoch:    uint64(status.ActiveEpoch),
	Orders:   msgBook,
}
```